mlir/lib/Dialect/MIOpen/Transforms/AffineTransforms.cpp

```diff
@@ -108,7 +108,8 @@ AffineMap AffineTransforms::buildIndexAffineMap(miopen::TransformOp op) {
           auto srcDim = srcDimAttr.getValue()[j].cast<IntegerAttr>().getInt();
           affExprsMap.insert({srcDim, expr});
         }
-      } else if (transformAttr.getValue() == "UnMerge") {
+      } else if ((transformAttr.getValue() == "UnMerge2") ||
```

Comment:
@ltqin the affine map for `UnMerge` and `UnMerge2` seems to be identical?

Reply:
computing diff is different, the diff of UnMerge is Embed, not UnMerge